### PR TITLE
Port test for ppc

### PR DESCRIPTION
**Pull request by sathnaga**
This patches tries to port this test to work in powerpc
host/guest and cleaned stress inside guest during test end.

Signed-off-by: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>

### CODE REVIEW

File: libvirt/tests/src/libvirt_bench/libvirt_bench_vcpu_hotplug.py [hunk 1]
```diff
@@ -58,7 +59,7 @@ def run(test, params, env):
         session.cmd("dmesg -c")
         for i in range(test_times):
             # 1. Add vcpu
-            add_result = libvirt.hotplug_domain_vcpu(vm_name,
+            add_result = libvirt.hotplug_domain_vcpu(vm,
```
[the remark this answers is not in the document]
Comment (sathnaga) [reply libvirt/tests/src/libvirt_bench/libvirt_bench_vcpu_hotplug.py]: This changes it, https://github.com/avocado-framework/avocado-vt/pull/1473

**Comment by sathnaga** [comment]
Dependancy PR: https://github.com/avocado-framework/avocado-vt/pull/1473

File: libvirt/tests/src/libvirt_bench/libvirt_bench_vcpu_hotplug.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (balamuruhans) [reply libvirt/tests/src/libvirt_bench/libvirt_bench_vcpu_hotplug.py]: Got it

**Comment by sathnaga** [comment]
Dependancy: https://github.com/avocado-framework/avocado-vt/pull/1488

**Pull request merged**